### DISCUSSION TITLE
使用Python，'AdvancedDict'关键字找不到，看了下，option也没有设置这个的东西，麻烦作者帮我看下

### DISCUSSION THREAD

**Discussion by Greeay7** [hect0x7/JMComic-Crawler-Python#305] [Q&A] [answered]
def parse_api_to_search_page(cls, data: AdvancedDict) -> JmSearchPage:
NameError: name 'AdvancedDict' is not defined

**Comment by hect0x7** [comment, marked as answer]
更新一下commonX

pip install commonX -U

**Comment by Greeay7** [comment]
好了，谢谢啦，不过我看
Installing collected packages: commonX
  Attempting uninstall: commonX
    Found existing installation: commonX 0.6.4
    Uninstalling commonX-0.6.4:
      Successfully uninstalled commonX-0.6.4
Successfully installed commonX-0.6.33
是新版本有问题是吗？

**Reply by hect0x7** [reply]
是旧版本有问题，0.6.33>0.6.4
